pkg/sql/ir_generator.go

```diff
@@ -127,6 +137,31 @@ func generateTrainStmtByModel(slct *parser.SQLFlowSelectStmt, connStr, cwd, mode
 	return generateTrainStmtWithInferredColumns(slctWithTrain, connStr)
 }
 
+func verifyTrainStmt(trainStmt *ir.TrainStmt, db *database.DB) error {
+	trainFields, e := verify(trainStmt.Select, db)
+	if e != nil {
+		return e
+	}
+
+	for _, fc := range trainStmt.Features {
+		for _, field := range fc {
+			for _, fm := range field.GetFieldDesc() {
+				name := fm.Name
+				_, ok := trainFields.get(name)
+				if !ok {
+					return fmt.Errorf("Feature field not exist in database: %s", name)
```

Comment:
1. Feature field **doesn't** exist
2. Error strings should not be capitalized https://github.com/golang/go/wiki/CodeReviewComments#error-strings